### DISCUSSION TITLE
Connection Refused Error

### DISCUSSION THREAD

**Discussion by Faleddo** [planetscale/discussion#363] [Q&A] [answered]
I randomly got this error message when using planetscale "Unable to connect to the database. Main connection [MySQLi]: unavailable: dial tcp 10.197.119.188:15999: connect: connection refused". Whats wrong with my config? It not happened everytime but just randomly can't connect

**Comment by orware** [comment, marked as answer]
Hello @Faleddo,

It's unclear from the error message only what could be causing this particular issue for your database, especially if it is occurring sporadically.

If it continues to occur, please reach out via: https://support.planetscale.com/ so that we may ask further questions and troubleshoot the situation together with you.
